package.json

```diff
@@ -49,7 +49,17 @@
     "appId": "com.symphony.electron-desktop",
     "mac": {
       "target": "dmg",
-      "category": "public.app-category.business"
+      "category": "public.app-category.business",
+      "extendInfo": {
```

Comment:
I don't think we'll need this since we set the minimum version via the Chrome flags. Let's remove this for now and add it later only if required.